examples/v2/chat2.nim

```diff
@@ -234,10 +233,21 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
   if conf.swap:
     node.mountSwap()
 
-  if conf.storenode != "":
+  if (conf.storenode != "") or (conf.store == true):
     node.mountStore()
 
-    node.wakuStore.setPeer(parsePeerInfo(conf.storenode))
+    var storenode: string
+
+    if conf.storenode != "":
+      storenode = conf.storenode
```

Comment:
Not sure if there is a format checking in the `parsePeerInfo` proc,  but I'd check whether the passed address is valid and if not set the store node randomly